src/test/ui/associated-types/defaults-cyclic-pass.rs

```diff
@@ -0,0 +1,22 @@
+// compile-pass
```

Comment:
change the test name from `defaults-cyclic-pass` - there is no cycle involved here

Reply:
Or `defaults-in-other-trait-items-pass`